src/renderer/components/settings/Preferences.vue

```diff
@@ -22,7 +22,7 @@
       <div class="row">
         <div class="column">
           <div class="label">{{$t('network')}}</div>
-          <aph-select :light="true" :options="networks" :initialValue="selectedNetwork" v-model="selectedNetwork" :allow-empty-value="false"></aph-select>
+          <aph-select :light="true" :options="networks" :isDisabled="isSystemAssetWithdrawInProgress" :initialValue="selectedNetwork" v-model="selectedNetwork" :allow-empty-value="false"></aph-select>
```

Comment:
In addition to just disabling the control, we need to show some sort of error message if they attempt to click to switch network while the withdraw is in progress.

Reply:
Probably should name the method `shouldDisableNetworkSelection`, since it will check other conditions soon also such as if a claim operation is still completing.

Reply:
The message is delivered to the user in the form of a 'tooltip'. When hover they will get the message indicating why it is locked.